packages/@okta/i18n/src/properties/login_cs.properties

```diff
@@ -922,10 +925,11 @@ oie.email.authenticator.description = Ov\u011B\u0159it pomoc\u00ED odkazu nebo k
 oie.email.mfa.title = Ov\u011B\u0159it pomoc\u00ED e-mailu
 oie.email.verify.primaryButton = Odeslat e-mail
 # {0} is a user's email address
-oie.email.verify.subtitle = Ov\u011B\u0159en\u00ED pomoc\u00ED e-mailov\u00E9ho odkazu nebo zad\u00E1n\u00EDm k\u00F3du bylo zasl\u00E1no spole\u010Dnosti <$1>{0}</$1>
+oie.email.verify.subtitleWithEmailAddress = Ov\u011B\u0159en\u00ED pomoc\u00ED e-mailov\u00E9ho odkazu nebo zad\u00E1n\u00EDm k\u00F3du bylo zasl\u00E1no spole\u010Dnosti <$1>{0}</$1>
+oie.email.verify.subtitleWithoutEmailAddress = Ov\u011B\u0159te pomoc\u00ED e-mailov\u00E9ho odkazu nebo zadejte k\u00F3d zaslan\u00FD na v\u00E1\u0161 e-mail.
 # {0} is a user's email address
 oie.email.verify.alternate.magicLinkToEmailAddress = Na adresu <$1>{0}</$1> byl e-mailem odesl\u00E1n jedine\u010Dn\u00FD odkaz (magic link). 
-oie.email.verify.alternate.magicLinkToYourEmail = Na <$1>v\u00E1\u0161 e-mail</$1> byl e-mailem odesl\u00E1n jedine\u010Dn\u00FD odkaz (magic link). 
+oie.email.verify.alternate.magicLinkToYourEmail = Na v\u00E1\u0161 e-mail byl e-mailem odesl\u00E1n jedine\u010Dn\u00FD odkaz (magic link). 
```

Comment:
Checking to see if it is ok to have this (magic link) text in english?

Reply:
looks like we had it in the previous version, so should be ok

Reply:
English loanword is usually okay with technical terms. I'll make a note to see if a localized version works better.